src/HotChocolate/PersistedQueries/src/PersistedQueries.Redis/Extensions/HotChocolateRedisPersistedQueriesServiceCollectionExtensions.cs

```diff
@@ -23,9 +23,12 @@ public static class HotChocolateRedisPersistedQueriesServiceCollectionExtensions
         /// A factory that resolves the redis database that
         /// shall be used for persistence.
         /// </param>
+        /// <param name="queryExpiryTimeSpan">
+        /// A timespan after that a query will be removed from the cache.
+        /// </param>
         public static IServiceCollection AddRedisQueryStorage(
             this IServiceCollection services,
-            Func<IServiceProvider, IDatabase> databaseFactory)
+            Func<IServiceProvider, IDatabase> databaseFactory, TimeSpan? queryExpiryTimeSpan = null)
```

Comment:
Can you do a line break here to align with the other arguments?

Reply:
sure, haven't looked closely